docs/deep-dives/data/preenrollment_bias.md

```diff
@@ -0,0 +1,116 @@
+
+## Retrospective A/A tests
+
+User behavior tends to be consistent over time. We've found week-to-week correlations of up to 80% for our key guardrail metrics. Given this strong correlation, we can look for evidence of imbalance during the _pre-experiment period_. We've been using the term preenrollment bias, though [others](https://www.statsig.com/blog/pre-experiment-bias-detection-statsig) use pre-experiment bias.
+
+During the pre-experiment period, our experimental cohorts should have no statistically significant difference across the dimensions (metrics) of interest. If there are statistically significant differences during the pre-experiment period, this is evidence of bias. This technique is called a [Retrospective A/A test](https://www.microsoft.com/en-us/research/articles/patterns-of-trustworthy-experimentation-pre-experiment-stage/).
+
+We now automatically run Retrospective A/A tests for all Nimbus experiments to test for imbalance in guardrail metrics. These can be found alongside the other statistical results in the [Jetstream data products](https://docs.telemetry.mozilla.org/datasets/jetstream.html#statistics-tables).
+
+In short, you can find them in the `moz-fx-data-experiments.mozanalysis.statistics_<slug>_preenrollment_<period>_1` where `<slug>` is the (snake case) experiment slug or ID (can be found using the Experimenter UI). We run analyses over 2 periods: the week prior to enrollment (`<period>` = `week`) and the 28-day period prior (`<period>` = `days_28`).
+
+For example:
+
+```sql
+SELECT *
+FROM `moz-fx-data-experiments.mozanalysis.statistics_<slug>_preenrollment_week_1`
+WHERE 1=1
+  AND comparison = 'relative_uplift'
+  AND comparison_to_branch = 'control'
+  AND statistic != 'deciles'
+  AND analysis_basis = 'exposures'
+ORDER BY metric, branch, statistic
+```
+
+## Covariate Adjustment & CUPED
+
+What if the Retrospective A/A test flags evidence of an imbalance? Should we have to discard that metric from our analysis completely? Luckily, there exist techniques to adjust for pre-experiment information. The most common/popular of these is [CUPED](https://www.statsig.com/blog/cuped). We have implemented a CUPED-like technique using linear models.
+
+Inferences for the [average treatment effect](https://en.wikipedia.org/wiki/Average_treatment_effect) (ATE) are most commonly calculated by computed the average (mean) in each treatment branch and then computing the difference. However, this can also be calculated using linear models.
+
+As an example, we can fit a model of the form:
+
+$$y_i = \beta_0 + \beta_t t_i$$
```

Comment:
Neat! Didn't know you could do this in markdown.

Reply:
You can if you add some packages!